content/en/blog/2022/jaeger-native-otlp/index.md

```diff
@@ -0,0 +1,108 @@
+languages.
+
+This causes challenges for both Jaeger users and OpenTelemetry maintainers:
+
+1.  **Confusing Choices**
+
+    Currently, users are faced with a choice of exporter (Jaeger or OTLP), and
+    this can be a source of confusion. A user might be inclined, when exporting
+    telemetry to Jaeger, to simply choose the Jaeger exporter because the name
+    matches (even though Jaeger now actively encourages the use of OTLP).
+
+    If we can eliminate this potentially confusing choice, we can improve the
+    user experience and continue standardizing on a single interoperable
+    protocol. We love it when things "just work" out of the box!
+
+2.  **Maintenance and duplication**
+
+    Because the Jaeger client libraries are now archived, they will not receive
+    updates (including security patches). To continue properly supporting Jaeger
+    client exporters, OpenTelemetry authors would be required to re-implement
+    some of the functionality it had previously leveraged from the Jaeger
+    clients.
+
+    Now that Jaeger supports OTLP, this feels like a step backwards: It results
+    in an increased maintenance burden with very little benefit.
+
+Now that Jaeger supports OTLP, this feels like a step backwards: It results in
+an increased maintenance burden with very little benefit.
+
+### User Impact
+
+The proposal is to deprecate the following exporters from OpenTelemetry in favor
+of using native OTLP into Jaeger:
+
+- Jaeger Thrift over HTTP
+- Jaeger Protobuf via gRPC
+- Jaeger Thrift over UDP
+
+In addition to application configuration changes, there could be other
+architectural considerations. HTTP and gRPC should be straightforward
+replacements, although it may require exposing ports 4317 and 4318 if they are
+not already accessible.
+
+Thrift over UDP implies the use of the
+[Jaeger Agent](https://www.jaegertracing.io/docs/1.24/architecture/#agent).
+Users with this deployment configuration will need to make a slightly more
+complicated change, typically one of the following:
+
+1. Direct ingest. Applications will change from using Thrift+UDP to sending OTLP
+   traces directly to their `jaeger-collector` instance. This may also have
+   sampling implications.
+2. Replacing the Jaeger Agent with a sidecar
+   [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector)
+   instance. This could have sampling implications and requires changes to your
+   infrastructure deployment code.
+
+## Intent to Deprecate - We'd Like Your Feedback!
+
+In order to better support users and the interop between OpenTelemetry and
+Jaeger, we intend to deprecate and eventually remove support for Jaeger client
+exporters in OpenTelemetry.
+
+We would like your feedback! We want to hear from users who could be impacted by
+this change. To better make a data-informed decision,
+[we have put together a short 4-question survey](https://forms.gle/aUuJg5DQwNzncJ4s8).
```

Comment:
the survey has 4-questions, but there are 5 bullet points with questions below, maybe survey question drift?

Reply:
Yeah, I had the list here before the survey was created....so they _never_ aligned exactly. This was intended to be talking/consideration points for folks chiming in on the PR....but with the survey in place it might read weird now. But it might read weird now. 

Might be more useful if it was moved to the bottom? Or I could remove it, or just have it match the survey (which is terser than this list).

Reply:
I'd vote to **remove** this list here (folks can always view the survey questions) of, if you keep them, ensure that they align.

Reply:
Ok I agree, especially with the additional user-impact section I think it's not really necessary. Will remove it.